packages/myst-to-typst/src/math.ts

```diff
@@ -55,16 +58,19 @@ export function resolveRecursiveCommands(plugins: MathPlugins): MathPlugins {
 }
 
 const math: Handler = (node, state) => {
-  const value = texToTypst(node.value);
+  const { value, macros } = texToTypst(node.value);
+  macros?.forEach((macro) => state.useMacro(macro));
+  const { identifier: label } = normalizeLabel(node.label) ?? {};
```

Comment:
We should update `normalizeLabel` to not allow spaces at all - these break typst and seem funny for other contexts.

Reply:
I think we can turn this into a bug and get this one out?

Reply:
https://github.com/jupyter-book/mystmd/issues/1429